tensorflow/compiler/jit/extract_outside_compilation_pass.cc

```diff
@@ -1803,8 +1805,9 @@ TF_ATTRIBUTE_NOINLINE Status ExtractOutsideCompilationForFuncCallNode(
     if (e->IsControlEdge()) {
       continue;
     }
-
-    TF_RET_CHECK(e->dst_input() >= 0 && e->dst_input() < inputs.size());
+    
+    const int check_condtion1 = e->dst_input() < inputs.size(); 
```

Comment:
Let's use a descriptive name here instead of `check_condition1`

Reply:
updated variable name.